tests/integration/helpers/helpers.go

```diff
@@ -217,3 +218,16 @@ func WaitForPort(fromContainer, host string, port int, timeout time.Duration) er
 
 	return fmt.Errorf("address %s:%d is not reachable after %v: %v", host, port, timeout, err)
 }
+
+// AssertReceivedErrors check if at least one the log lines provided contains the given message.
+func AssertReceivedErrors(t *testing.T, msg string, errLog ...string) {
+	assert.GreaterOrEqual(t, len(errLog), 1)
+
+	for _, line := range errLog {
+		if strings.Contains(line, msg) {
+			return
+		}
+	}
+
+	assert.Failf(t, fmt.Sprintf("Expected to find the following error message: %s", msg), "but got %s", errLog)
```

Comment:
```suggestion
	assert.Failf(t, "Expected to find the following error message: %q, but it was not found in %v", msg, errLog)
```
Nit: we don't need an additional `fmt.Sprintf`, do we?

Reply:
resolved in this [commit](https://github.com/spathlavath/nri-mysql/pull/27/commits/06b1a98688bb4adacb2065b68e17dce9d225fe1f)

Reply:
When I'm using the below assert statement. The logs don't show the values of msg, errLog in the output here is an [example](https://github.com/spathlavath/nri-mysql/actions/runs/12864914029/job/35864511894?pr=27#:~:text=578-,Error%3A%20%20%20%20%20%20%09Expected%20to%20find%20the%20following%20error%20message%3A%20%25q%2C%20but%20it%20was%20not%20found%20in%20%25v,-579)

> assert.Failf(t, "Expected to find the following error message: %q, but it was not found in %v", msg, errLog)

Reverted back to the previous assert statement with `fmt.Sprintf` in this [commit](https://github.com/spathlavath/nri-mysql/pull/27/commits/d841a30c61b498984828dd8def47410e19535836)